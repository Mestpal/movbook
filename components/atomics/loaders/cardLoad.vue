<template>
  <v-flex
    xs12
    sm6
    md4
    lg3
    xl3
  >
    <v-skeleton-loader
      type="card"
    />
  </v-flex>
</template>
