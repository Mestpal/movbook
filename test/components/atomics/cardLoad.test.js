
import Vue from 'vue'
import Vuetify from 'vuetify'
import Vuex from 'vuex'
import { mount, createLocalVue } from '@vue/test-utils'

import cardLoad from '@/components/atomics/cardLoad'

Vue.use(Vuetify)
Vue.use(Vuex)
const localVue = createLocalVue()

describe('cardLoad.vue', () => {
  let wrapper

  beforeEach(() => {
    wrapper = mount(cardLoad, {
      localVue
    })
  })

  it('reders the cardLoad', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
