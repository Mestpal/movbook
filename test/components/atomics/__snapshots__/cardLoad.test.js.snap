// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cardLoad.vue reders the cardLoad 1`] = `
"<div class=\\"flex xs12 sm6 md4 lg3 xl3\\">
  <div aria-busy=\\"true\\" aria-live=\\"polite\\" role=\\"alert\\" class=\\"v-skeleton-loader v-skeleton-loader--is-loading theme--light\\">
    <div class=\\"v-skeleton-loader__card v-skeleton-loader__bone\\">
      <div class=\\"v-skeleton-loader__image v-skeleton-loader__bone\\"></div>
      <div class=\\"v-skeleton-loader__card-heading v-skeleton-loader__bone\\">
        <div class=\\"v-skeleton-loader__heading v-skeleton-loader__bone\\"></div>
      </div>
    </div>
  </div>
</div>"
`;
